melosys-eessi-app/src/main/java/no/nav/melosys/eessi/identifisering/IdentifiseringKontrollService.java

```diff
@@ -20,14 +20,15 @@
     private final PersonFasade personFasade;
     private final EuxService euxService;
     private final PersonSokMetrikker personSokMetrikker;
+    private final static int OVERSTYREKONTROLLVERSJON = 2;
```

Comment:
Er vi sikre på at det er 2 som vil gjelde?
Hva hvis ID og fordeling skriver en kommentar i oppgaven før de er ferdige med behandling?

Reply:
Jeg har overstyrtversjon = 2, og i sjekken så sjekker vi mot > overstyrtversjon. Det er for å unngå nettopp det du skriver over.

Reply:
Kan endre koden slik at det er mer lesbart ihht til navn